rasa/shared/nlu/training_data/training_data.py

```diff
@@ -60,6 +60,34 @@ def __init__(
 
         self._fill_response_phrases()
 
+    @staticmethod
+    def _load_lookup_table(lookup_table: Dict[Text, Any]) -> Dict[Text, Any]:
+        """Loads the actual lookup table from file if there is a file specified.
+
+        Checks if the specified lookup table contains a filename in
+        `elements` and replaces it with actual elements from the file.
+        Returns the unchanged lookup table otherwise.
+
+        Params:
+            lookup_table: A lookup table.
+
+        Returns:
+            Updated lookup table where filenames are replaced with the contents of these files.
+        """
+        elements = lookup_table["elements"]
+        potential_file = elements if isinstance(elements, str) else elements[0]
+
+        if Path(potential_file).is_file():
+            try:
+                lookup_table["elements"] = rasa.shared.utils.io.read_file(
+                    potential_file, "utf8"
```

Comment:
utf-8 is the default
```suggestion
                    potential_file
```